### ISSUE TITLE
Issue with nginx-ingress-gke

### ISSUE THREAD

**Issue by paulwoelfelcpl** [GoogleCloudPlatform/community#891]
# Issue Description

The tutorial does not include the `controller.publishService.enabled` flag for installation of nginx-ingress. Without this flag a wrong IP will be published in the status of the ingress resource. 

This is documented in [issue 4092 on nginx-ingress](https://github.com/kubernetes/ingress-nginx/issues/4092). See also the [nginx-ingress Readme](https://github.com/helm/charts/blob/master/stable/nginx-ingress/README.md#configuration).

# Suggested solution

```bash
helm install --name nginx-ingress stable/nginx-ingress --set rbac.create=true --set controller.publishService.enabled=true
```

**Comment by ToddKopriva** [comment]
Thanks, Paul.

If you submit a pull request for the tutorial with that fix, I'll review it and merge it.

**Comment by ToddKopriva** [comment]
fixed by pull request #894
